src/azure-cli/azure/cli/command_modules/sql/_help.py

```diff
@@ -1282,6 +1282,35 @@
     text: az sql midb log-replay wait --deleted --resource-group mygroup --managed-instance myinstance --name mymanageddb
 """
 
+helps['sql midb ledger-digest-uploads'] = """
```

Comment:
```suggestion
helps['sql midb ledger-digest-upload'] = """
```

Reply:
we use same cmd in azure sql also. So, using the same to maintain the consistency between azure sql and MI.
https://learn.microsoft.com/en-us/cli/azure/sql/db/ledger-digest-uploads?view=azure-cli-latest#az-sql-db-ledger-digest-uploads-enable